Now you know the problem problaly in the build_target process
Now just try ur own __call__ method

That is funny lmao
the __call__ function is also incorrect

I change the loss_obj from `score_iou` -> `1`, and i get the same result.
So i think the problem is in `score_iou`

And in fact, i even donot know the meaning of `score_iou`

OK, fine, i just give it up now

For now, i think the `score_iou` is the error stem from